common/lanelet2_extension/lib/autoware_traffic_light.cpp

```diff
@@ -28,8 +28,17 @@ namespace lanelet
 {
 namespace autoware
 {
+
+#if __cplusplus < 201703L
```

Comment:
I think this pertains to the issue you and I discussed in the lab, but probably worth adding a line or two of comments explaining exactly why this C++ version check is needed.

Reply:
Done